packages/google-cloud-storage-control/.repo-metadata.json

```diff
@@ -10,8 +10,8 @@
     "library_type": "GAPIC_AUTO",
     "repo": "googleapis/google-cloud-python",
     "distribution_name": "google-cloud-storage-control",
-    "api_id": "control.googleapis.com",
+    "api_id": "storage.googleapis.com",
     "default_version": "v2",
-    "codeowner_team": "",
-    "api_shortname": "storage"
+    "codeowner_team": "@googleapis/cloud-storage-dpe",
```

Comment:
We haven't started using the `codeowner_team` in this repository yet. Please reach out internally if we need to explicitly add `googleapis/cloud-storage-dpe` as a code owner.